src/targets/services/service-worker.js

```diff
@@ -0,0 +1,8 @@
+import { precacheAndRoute } from 'workbox-precaching'
+
+precacheAndRoute(self.__WB_MANIFEST)
+
+// I choose https://developer.chrome.com/docs/workbox/the-ways-of-workbox/#workbox-cli
+// but we can integrate Workbox another way
+// using Webpack build process, thanks to generateSW or injectManifest
+// this can be done after this story's validation
```

Comment:
Since this file can be used for browser but also for intent (we will see that later), we can't put it in /target/browser. So let's add it somewhere in ./src/. I imagine that someday the service worker will grow, so it's not absurd to create a ./src/serviceWorker folder ?